#Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/03
#Thomas Hoffmann, EMBL Heidelberg, 2021/03
#Switch off build of voltools, mpi4py, and NFFT in pytomc/compile.py in order to use them from dependencies.
diff -ru disable_external_build/pytomc/compile.py ori/pytomc/compile.py
diff -ru ori/pytomc/compile.py disable_external_build/pytomc/compile.py
--- ori/pytomc/compile.py	2021-01-18 22:40:34.000000000 +0100
+++ disable_external_build/pytomc/compile.py	2021-03-11 16:46:30.366174266 +0100
@@ -193,10 +193,10 @@
 print(setflags_line)
 print('')
 
-nosh = False # you can choose not to compile the SH Alignment library
-nompi4py = False # you can choose not to compile the mpi4py library
-nonfft = False # you can choose not to compile the NFFT library
-novoltools = False
+nosh = True # you can choose not to compile the SH Alignment library
+nompi4py =True # False # you can choose not to compile the mpi4py library
+nonfft =True # False # you can choose not to compile the NFFT library
+novoltools =True # False
 
 
 if phony_target == "swig":
